pyproject.toml

```
@@ -201,6 +204,7 @@ minimal-server = [
     "canonicaljson",
     "jinja2",
     "jmespath",
+    "json_merge_patch",
```

Here too, we need `jsonpatch`.

```suggestion
    "json_merge_patch",
    "jsonpatch",
```